Octokit/Models/Request/RequestParameters.cs

```diff
@@ -49,6 +49,9 @@ static List<PropertyParameter> GetPropertyParametersForType(Type type)
             Justification = "GitHub API depends on lower case strings")]
         static Func<PropertyInfo, object, string> GetValueFunc(Type propertyType)
         {
+            // get underlying type if nullable
+            propertyType = Nullable.GetUnderlyingType(propertyType) ?? propertyType;
```

Comment:
Ah fair enough, I must admit I just quickly put the nullable properties on the request object and ran the existing integration tests which passed...  From what you're saying we might need more tests since the current ones didn't pick up the problem you highlighted...